src/Timeline/components/EventBlock.js

```diff
@@ -56,14 +55,10 @@ const EventBlock = ({
   return (
     <>
       <Date>{date}</Date>
-      <TimelineDivider
-        isFirst={isFirst}
-        isLast={isLast}
-        isFirstGroupMajorOnly={isFirstGroupMajorOnly}
-      >
+      <TimelineDivider isFirst={isFirst} isLast={isLast}>
         {description && (
           <Marker
-            isFirstGroupMajorOnly={isFirstGroupMajorOnly}
+            showMinorEventUI={!showMinorEventUI}
```

Comment:
Should be renamed `hideMinorEventUI` since we're inverting?

Reply:
I think instead i need to invert when it's used within Marker - but it actually looks like it's not needed at all there so will remove it.
 I'm inverting showMinorEventUI on the EventBlock prop too, I'll change that and invert within the component

Reply:
Yeah I think that's correct - either it's `PREACTIVE` or `PENDING_CANCEL` and we have an expected completion event or it's `ACTIVE` or `CEASED` in which case the last event will be both major and terminal.

Reply:
Will see if we can get this confirmed before removing the prop. Might still be worth leaving in incase the scenario pops up in the future